src/nvim/globals.h

```diff
@@ -1123,7 +1123,7 @@ EXTERN char_u e_noalt[] INIT(= N_("E23: No alternate file"));
 EXTERN char_u e_noabbr[] INIT(= N_("E24: No such abbreviation"));
 EXTERN char_u e_nobang[] INIT(= N_("E477: No ! allowed"));
 EXTERN char_u e_nogvim[] INIT(= N_(
-        "E25: GUI cannot be used: Not enabled at compile time"));
+        "E25: Nvim does not have built-in GUI"));
```

Comment:
It is used by *every* po file to lookup translations. 

```
$ grep -r 'E25: GUI cannot be used: Not enabled at compile time'
nvim/po/zh_CN.UTF-8.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
nvim/po/pl.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
nvim/po/fr.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
<...snip...>
nvim/po/uk.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
nvim/po/eo.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
nvim/po/pl.UTF-8.po:msgid "E25: GUI cannot be used: Not enabled at compile time"
nvim/globals.h:        "E25: GUI cannot be used: Not enabled at compile time"));
```

I guess we should be sure the new message is what we want :)